default.nix

```diff
@@ -43,6 +43,15 @@ stdenv.mkDerivation rec {
     # CoqIDE dependencies
     ocamlPackages.lablgtk
 
+  ] else []) ++ (if buildDoc then [
+
+    # Sphinx doc dependencies
+    ocamlPackages.lablgtk
```

Comment:
Is it because CoqIDE is built whenever the documentation is built? We should remember to fix that.